files/default/neutron-ha-tool.py

```diff
@@ -737,10 +737,21 @@ def target_agent_list(agent_list, agent_type, exclude_agent_host):
     :param exclude_agent_host: hostname of agent to exclude from the list
 
     """
+    agent_info = [
```

Comment:
Shorter does not imply clearer; in fact IMHO it's the exact opposite in this case: this function is dealing with multiple agents, so it's ambiguous which agent `agent_info` refers to.  The only way to understand which agent it refers to is to read the code which assigns to it.  (I'm not sure exactly what you mean by "not from who"?)

In contrast, if it was called `excluded_agent_info` then it would be self-evident, because it would obviously be referring to the docstring which says "Return a list of agents that are alive, **excluding** the one we want to migrate from".